discovery-infra/tests/base_test.py

```diff
@@ -26,6 +26,8 @@ def nodes(self, setup_node_controller):
         controller = setup_node_controller
         nodes = Nodes(controller, env_variables["private_ssh_key_path"])
         nodes.set_correct_boot_order(start_nodes=False)
+        nodes.run_for_all_nodes("reset_cpu_cores")
```

Comment:
missed that - this should be running after the yield (both this and the next line)
no?
save orig, run tests, do changes, then back to orig